Open Judge System/Web/OJS.Web/Web.config

```diff
@@ -141,186 +150,186 @@
+			</dependentAssembly>
+		</assemblyBinding>
+	</runtime>
+	<entityFramework>
+		<defaultConnectionFactory type="System.Data.Entity.Infrastructure.LocalDbConnectionFactory, EntityFramework">
+			<parameters>
+				<parameter value="v11.0" />
+			</parameters>
+		</defaultConnectionFactory>
+		<providers>
+			<provider invariantName="System.Data.SqlClient" type="System.Data.Entity.SqlServer.SqlProviderServices, EntityFramework.SqlServer" />
+		</providers>
+	</entityFramework>
+	<glimpse defaultRuntimePolicy="On" endpointBaseUri="~/Glimpse.axd">
+		<runtimePolicies>
+			<ignoredTypes>
+				<add type="Glimpse.AspNet.Policy.LocalPolicy, Glimpse.AspNet" />
+			</ignoredTypes>
+		</runtimePolicies>
+	</glimpse>
+	<system.webServer>
+		<handlers>
+			<add name="UrlRoutingHandler" path="/Users/*" type="System.Web.Routing.UrlRoutingHandler" verb="GET" />
+			<add name="Glimpse" path="glimpse.axd" verb="GET" type="Glimpse.AspNet.HttpHandler, Glimpse.AspNet" preCondition="integratedMode" />
+		</handlers>
+		<staticContent>
+			<remove fileExtension=".woff" />
+			<mimeMap fileExtension=".woff" mimeType="application/x-font-woff" />
+		</staticContent>
+		<directoryBrowse enabled="false" />
+		<rewrite>
+			<rules />
+		</rewrite>
+		<validation validateIntegratedModeConfiguration="false" />
+		<modules>
+			<add name="Glimpse" type="Glimpse.AspNet.HttpModule, Glimpse.AspNet" preCondition="integratedMode" />
+			<add name="ErrorLog" type="Elmah.ErrorLogModule, Elmah" preCondition="managedHandler" />
+			<add name="ErrorMail" type="Elmah.ErrorMailModule, Elmah" preCondition="managedHandler" />
+			<add name="ErrorFilter" type="Elmah.ErrorFilterModule, Elmah" preCondition="managedHandler" />
+		</modules>
+	</system.webServer>
+	<location path="Administration">
+		<system.web>
+			<httpRuntime maxRequestLength="1048576" executionTimeout="600" />
+		</system.web>
+		<system.webServer>
+			<security>
+				<requestFiltering>
+					<requestLimits maxAllowedContentLength="1073741824" />
+				</requestFiltering>
+			</security>
+		</system.webServer>
+	</location>
+	<elmah>
+		<security allowRemoteAccess="true" />
+		<errorLog type="Elmah.XmlFileErrorLog, Elmah" logPath="D:\OjsFiles\Logs\Elmah" />
+	</elmah>
+	<system.codedom>
+		<compilers>
+			<compiler language="c#;cs;csharp" extension=".cs" type="Microsoft.CodeDom.Providers.DotNetCompilerPlatform.CSharpCodeProvider, Microsoft.CodeDom.Providers.DotNetCompilerPlatform, Version=2.0.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" warningLevel="4" compilerOptions="/langversion:default /nowarn:1659;1699;1701" />
+			<compiler language="vb;vbs;visualbasic;vbscript" extension=".vb" type="Microsoft.CodeDom.Providers.DotNetCompilerPlatform.VBCodeProvider, Microsoft.CodeDom.Providers.DotNetCompilerPlatform, Version=2.0.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" warningLevel="4" compilerOptions="/langversion:default /nowarn:41008 /define:_MYTYPE=\&quot;Web\&quot; /optionInfer+" />
```

Comment:
Why those are changed? Maybe configure your IDE spacing settings?

Reply:
They are configured, but because of local conflict i copied the code from the Web.config in master, and apparently it didn't format it correctly